google-cloud-clients/google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/SubscriberStats.java

```diff
@@ -27,52 +27,52 @@
 
   @AutoValue
   // TODO: Finish implementation.
-  public abstract static class Stats {}
+  abstract static class Stats {}
```

Comment:
To confirm, everything in this class is only package private? Want to make sure we won't break any existing implementations.

Reply:
correct.  The top level class `SubscriberStats` is package private, so no one can access the `Stats` inner class from outside the package.